packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -231,12 +231,26 @@ class PerformanceModeRequestHandle {
 ///   priority and are executed in priority order according to a
 ///   [schedulingStrategy].
 mixin SchedulerBinding on BindingBase {
+  /// Override [_enableProfileFrame] in debug mode.
+  @visibleForTesting
+  static bool? debugOverrideEnableProfileFrame;
+
+  /// Whether to send frame events to DevTool
+  static bool get _enableProfileFrame {
+    bool? override;
+    assert(() {
+      override = debugOverrideEnableProfileFrame;
+      return true;
+    }());
+    return override ?? const bool.fromEnvironment('FLUTTER_PROFILE_FRAME', defaultValue: true);
+  }
```

Comment:
Because they have same assembly :)

https://godbolt.org/z/rs9hhfeMc

![image](https://user-images.githubusercontent.com/5236035/198157219-70041837-e708-425b-b443-0f69b791c671.png)

Reply:
I can remove the environment variable, see below.